library/std/src/primitive_docs.rs

```diff
@@ -478,8 +478,10 @@ mod prim_unit {}
 #[stable(feature = "rust1", since = "1.0.0")]
 mod prim_pointer {}
 
+#[doc(alias = "[;]")]
+#[doc(alias = "[T;N]")] // unfortunately, rustdoc doesn't have fuzzy search for aliases
+#[doc(alias = "[T; N]")]
```

Comment:
I'm not sure those aliases will be super useful... Maybe `[]`?

Reply:
@GuillaumeGomez `[]` is already used for slices: https://doc.rust-lang.org/std/?search=%5B%5D. Is it possible to have two different types with the same alias?

Reply:
Yup you can. :)

Reply:
Ok, I changed `[;]` to `[]`. I still think `[T; N]` is useful because it's used in the documentation a lot.

Reply:
Maybe keep `[;]` too then? XD